src/Operation/Partition.php

```diff
@@ -31,25 +31,29 @@ final class Partition extends AbstractOperation
     public function __invoke(): Closure
     {
         return
-            /**
-             * @param callable(T, TKey, Iterator<TKey, T>): bool ...$callbacks
-             *
-             * @return Closure(Iterator<TKey, T>): Generator<int, array{0: Closure(Iterator<TKey, T>): Generator<TKey, T>, 1: array{0: Iterator<TKey, T>}}>
```

Comment:
still missing this Closure return type which was removed

Reply:
I added them back despite the fact that they were not really required.